pymatgen/io/qchem/inputs.py

```diff
@@ -18,11 +18,8 @@
 
 from .utils import lower_and_check_unique, read_pattern, read_table_pattern
 
-__author__ = "Brandon Wood, Samuel Blau, Shyam Dwaraknath, Julian Self, Evan Spotte-Smith"
-__copyright__ = "Copyright 2018, The Materials Project"
-__version__ = "0.1"
-__email__ = "b.wood@berkeley.edu"
-__credits__ = "Xiaohui Qu"
```

Comment:
Probably Xiaohui should still get credit. His original work inspired what's now in pymatgen.

Reply:
I guess I'm not clear on the distinction between `__credits__` and `__authors__` (I might just be ignorant). Those seemed equivalent to me. Happy to revert though if `__credits__` implies origination or some other special status.

Reply:
I did make sure to add Xiaohui to the `__authors__` list for any file where he was listed in `__credits__`, in case that wasn't obvious